src/stories/sampleCardComponents/vitalCard.scss

```diff
@@ -0,0 +1,80 @@
+@use '../../common/scss/global.scss';
+@use '../../common/scss/mixins.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+@use '@kyndryl-design-system/shidoka-foundation/scss/mixins/visibility.scss';
+
+.vital-card {
+  &-title-label {
+    @include typography.type-body-02;
+    color: var(--kd-color-text-secondary);
+    margin-bottom: 8px;
+  }
+  &-title {
+    @include typography.type-body-02;
+    font-weight: 500;
+    margin-top: 0px;
+    margin-bottom: 0px;
+    color: var(--kd-color-text-primary);
+    // for large device after md size
+    @media (min-width: 42rem) {
+      @include typography.type-headline-07;
+      font-weight: 400;
+      margin-bottom: 4px;
+    }
+  }
+  &-title-div {
+    padding-right: 4px;
+  }
+
+  &-content-wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    @media (min-width: 42rem) {
+      display: block;
+    }
+  }
+  &-mobile-wrapper-subdiv {
+    display: flex;
+    align-items: center;
+    @media (min-width: 42rem) {
+      display: block;
+    }
+  }
+
+  &-category-text {
+    @include typography.type-ui-02;
+    color: var(--kd-color-text-primary);
+    margin-top: 0px;
+    margin-bottom: 0px;
+  }
+  &-subcategory-text {
+    @include typography.type-ui-03;
+    color: var(--kd-color-text-secondary);
+    margin-top: 0;
+    margin-bottom: 4px;
+  }
+  &-cat-subcat-text {
+    padding: 0px 8px;
+    @media (min-width: 42rem) {
+      padding: 0;
+    }
+  }
+  &-link {
+    margin-top: 0px;
+    @media (min-width: 42rem) {
+      margin-top: 10px;
+    }
+    &-text {
+      @media (max-width: 42rem) {
+        @include visibility.hidden;
```

Comment:
Wrong mixin. This sets display: none. You need "sr-only" instead.

Reply:
oh ok

Reply:
Done